e2e/pages/web-pages/campaigns/donation.page.ts

```diff
@@ -0,0 +1,220 @@
+    private readonly enSelectAmountSectionText = enLocalizationOneTimeDonation['step-labels'].amount;
+    // TODO Add these three IDs into the component (if possible) and update the test methods
+    private readonly donationAmount = this.allAmountsSelector + " #donationAmount";
+    private readonly feeAmount = this.allAmountsSelector + " #feeAmount";
+    private readonly totalChargedAmount = this.allAmountsSelector + " #totalChargedAmount";
+    // Grid navigation buttons localization
+    private readonly bgForwardButtonText = bgLocalizationOneTimeDonation.btns.next;
+    private readonly enForwardButtonText = enLocalizationOneTimeDonation.btns.next;
+    
+    // -> Personal profile section <-
+    private readonly buttonsContainer = ".MuiTabs-flexContainer button";
+    private readonly bgDonateAnonymouslyText = bgLocalizationOneTimeDonation['second-step']['donate-anonymously'];
+    private readonly enDonateAnonymouslyText = enLocalizationOneTimeDonation['second-step']['donate-anonymously'];
+    private readonly inputRootSelector = ".MuiInputBase-root";
+    private readonly donateAnonymouslyEmailField = this.inputRootSelector + " input[name='personsEmail']";
+     // Section labels
+     private readonly bgPersonalProfileSectionText = bgLocalizationOneTimeDonation['step-labels']['personal-profile'];
+     private readonly enPersonalProfileSectionText = enLocalizationOneTimeDonation['step-labels']['personal-profile'];
+
+    // -> Send a wish section <-
+    // Section labels
+    private readonly sendAWishField = this.inputRootSelector + " textarea[name='message']";
+    private readonly bgSendAWishSectionText = bgLocalizationOneTimeDonation['step-labels'].wish;
+    private readonly enSendAWishSectionText = enLocalizationOneTimeDonation['step-labels'].wish;
+
+    // -> Payment <-
+    // Section labels
+    private readonly bgPaymentSectionText = bgLocalizationOneTimeDonation['step-labels'].payment;
+    private readonly enPaymentSectionText = enLocalizationOneTimeDonation['step-labels'].payment;
+    private readonly bgFinishButtonText = bgLocalizationOneTimeDonation.btns.end;
+    private readonly enFinishButtonText = enLocalizationOneTimeDonation.btns.end;
+    private readonly bgSuccessfulDonationTitle = bgLocalizationOneTimeDonation.success.title;
+    private readonly enSuccessfulDonationTitle = enLocalizationOneTimeDonation.success.title;
+
+    /**
+     * Is "Select amount" step active
+     */
+    async isSelectAmountStepActive(language: LanguagesEnum = LanguagesEnum.BG): Promise<boolean> {
+        if (language === LanguagesEnum.BG) {
+            return this.isStepActiveByLabelText(this.bgSelectAmountSectionText);
+        } else if (language === LanguagesEnum.EN) {
+            return this.isStepActiveByLabelText(this.enSelectAmountSectionText);
+        } else {
+            throw new Error("Language not found!");
+        }
```

Comment:
I think those ifs create unnecessary code duplication.
It would be better to have a localization map or even different configuration files per language. 
and then just say:

    return this.isStepActiveByLabelText(t(lang, 'select-amount-section-text'));